app/controllers/githook_controller.rb

```diff
@@ -38,9 +42,17 @@ def pull_master
   end
 
   def authenticate
-    flash[:error] = t("the_role.access_denied")
-    redirect_to(:back) unless (current_user) && (current_user.moderator?(:val))
+    flash[:error] = t('the_role.access_denied')
+    redirect_to(:back) unless user_signed_in? && current_user.admin?
   rescue ActionController::RedirectBackError
     redirect_to root_path
   end
+
+  def verify_signature(payload_body)
+    signature = 'sha1=' + OpenSSL::HMAC.hexdigest(
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.